packages/web/src/components/dynamic-image/DynamicImage.tsx

```diff
@@ -154,7 +158,28 @@ const DynamicImage = (props: DynamicImageProps) => {
     immediate
   ])
   return (
-    <div className={cn(styles.wrapper, wrapperClassName)} {...other} role='img'>
+    <Box
+      className={cn(styles.wrapper, wrapperClassName)}
+      {...other}
+      role='img'
```

Comment:
do we only want the &:before element when useBlur is true? would we want to create a "blur" style like `const blurCss = css({...})` and use it in various places? or maybe a BlurrableBox of sorts?

Reply:
yeah I was thinking about that. I'll see if I can make something reusable